File: main (4).py
'''

Welcome to GDB Online.
GDB online is an online compiler and debugger tool for C, C++, Python, Java, PHP, Ruby, Perl,
C#, OCaml, VB, Swift, Pascal, Fortran, Haskell, Objective-C, Assembly, HTML, CSS, JS, SQLite, Prolog.
Code, Compile, Run and Debug online from anywhere in world.
'''


def solution(lis):
    colours = {}
    p = 0
    for i in range(len(lis)):
        for j in range(len(lis[i])):
            if lis[i][j] in colours.keys():
                colours[lis[i][j]]+=1
            else:
                colours[lis[i][j]]=1
                
        
    print (colours.keys())
    print(colours)

obj1 = ["r", "g", "b"]
obj2 = ["bl", "y", "w"]
obj3 = ["r", "w", "p"]
obj4 = ["p", "y", "b"]
obj5 = ["c", "m", "ch"]
lis = [obj1, obj2, obj3, obj4, obj5]
solution(lis)

File: main (4).cpp
/******************************************************************************

                              Online C++ Compiler.
               Code, Compile, Run and Debug C++ program online.
Write your code in this editor and press "Run" button to compile and execute it.

*******************************************************************************/

#include <iostream>

using namespace std;

int main()
{
    int n= 11;
    for(int i=0; i<=n/2; i++){
        for(int j=0; j<i; j++){
            cout<<" ";
         
        }
        if(i==n/2){
            cout<<"*";
        }
        for(int k=i; k<(n/2); k++){
            cout<<"*";
        }
        for(int l=i; l<(n/2); l++){
            cout<<"*";
        }
        cout<<endl;
    }

    return 0;
}
